.travis.yml

```diff
@@ -26,7 +26,7 @@ install:
 script:
   - ./run-qa-checks
   - coverage run --source=openwisp_ipam runtests.py
-  - SAMPLE_APP=1 ./runtests.py
+  - SAMPLE_APP=1 ./runtests.py --parallel --keepdb
```

Comment:
We can replicate it in other modules as well, since SAMPLE_APP tests are not contributing to coverage anyway, what do you think?

Reply:
I see 2 seconds difference only, I would have expected more :(
So the position of the two lines does not interfere with coverage right?
I think we can proceed with it, I would just add a short comment explaining why we're doing it there and not in the other line, something like:
```
# SAMPLE tests do not influence coverage, so we can speed up tests with --parallel
```

Reply:
> I think we can proceed with it, I would just add a short comment explaining why we're doing it there and not in the other line, something like:

It is already mentioned in README, I assume people will look at README and only developers would peek in travis, shall I still add it here as well?

Reply:
the purpose of the comment is to indicate why the first command uses `coverage run --source=openwisp_ipam runtests.py` and the second one does not use coverage and has parallel, it can be shorter than the text I suggested.